tests/director/tests/formation_constraint_api_test.go

```diff
@@ -309,6 +327,15 @@ func TestListFormationConstraintsForFormationTemplate(t *testing.T) {
 	t.Logf("Attaching second constraint to formation template")
 	fixtures.AttachConstraintToFormationTemplate(t, ctx, certSecuredGraphQLClient, constraintSecond.ID, formationTemplate.ID)
 
+	// Assert the two constraints are attached only to the first formation template
+	t.Logf("Get formation template with name %q and id %q, and assert there are is one constraint attached to it", formationTemplate.Name, formationTemplate.ID)
+	formationTemplateOutput = fixtures.QueryFormationTemplateWithConstraints(t, ctx, certSecuredGraphQLClient, formationTemplate.ID)
+	assert.Equal(t, formationTemplateOutput.FormationConstraints, []graphql.FormationConstraint{*constraint, *constraintSecond})
```

Comment:
Mismatch between the log message and the assert, the log says one constraint but the assert check for two